api/proto/banyandb/database/v1/rpc.proto

```diff
@@ -354,10 +355,56 @@ message GroupRegistryServiceUpdateResponse {}
 
 message GroupRegistryServiceDeleteRequest {
   string group = 1;
+  bool dry_run = 2;
+  bool force = 3;
```

Comment:
The deletion process is asynchronous. What is the difference between forced and graceful deletion?